### DISCUSSION TITLE
Akses pengguna umum

### DISCUSSION THREAD

**Discussion by AlifAbdulRauf** [noplanalderson/simasjid#6] [Q&A]
Izin bang mau nanya, kalau mau lihat home semisalnya ada pengguna umum/masyarakat cara aksesnya bagaimana ya? Soalnya waktu saya coba langsung di arahkan ke login. Terimakasih bang 🙏

**Comment by noplanalderson** [comment]
Aplikasi simasjid ini tidak dirancang untuk diakses langsung oleh umum dan sudah tidak dikembangkan lagi karena berbagai hal.
Terima kasih.
